services/venice-controller/src/main/java/com/linkedin/venice/pushmonitor/PushMonitorUtils.java

```diff
@@ -170,6 +170,12 @@ public static ExecutionStatusWithDetails getDaVinciPushStatusAndDetails(
       }
       String statusDetail = statusDetailStringBuilder.toString();
       if (allInstancesCompleted) {
+        LOGGER.info(
```

Comment:
Just want to call it out here is that during the feature rollout, the push job might fail as follows:
1. All the nodes are emitting partition-level status.
2. During the deployment, a few nodes will stop emitting the partition-level status, but only emit version-level status.
3. This logic will check both version-level report and partition-level report, even all the nodes finish the ingestion, the partition-level status aggregation will fail as it will detect a list of offline nodes (partition-level report stops because of the deployment in the deployed nodes).

Ideally, we should ignore the nodes , which are emitting version-level status in partition-level status aggregation.

Does ^ make sense?
cc @sixpluszero

Reply:
@gaojieliu Good catch! Updated the diff to address this edge case.